### DISCUSSION TITLE
Weekly Meeting 9th Aug 2022

### DISCUSSION THREAD

**Discussion by madjin** [omigroup/omi-scripting-group#38] [Show and tell]
# Agenda

We're doing our developer hour / demo day today, folks are welcome to share what they been working on

- Adam has couple demos
  - Identity verification protocol
    - much simpler version of W3C DIDs
    - Can implement in few hours, simpler / more robust on distributed SSI
  - Digital good / license protocol
    - Way of recording digital assets that can work with existing web infra
    - Built on http, works fairly similar to domain names
  - Scene graph transmission protocol
    - distribute state of a scene across multiple clients (can't demo today)
- Robert has a small intro to lightmap work hes been doing (doing bulk at glTF meeting later this week)

**Comment by robertlong** [comment]
We also should take a quick minute at the end of the meeting to discuss splitting out the developer hour from the scripting group and changing the time to 1700 UTC across the board.

**Reply by robertlong** [reply]
Also should we take that route we should update the roles, calendar, etc.
